src/topology/metric_space/gromov_hausdorff.lean

```diff
@@ -144,30 +137,25 @@ lemma to_GH_space_eq_to_GH_space_iff_isometric {X : Type u} [metric_space X] [co
   to_GH_space X = to_GH_space Y ↔ nonempty (X ≃ᵢ Y) :=
 ⟨begin
   simp only [to_GH_space, quotient.eq],
-  assume h,
-  rcases h with ⟨e⟩,
+  rintro ⟨e⟩,
   have I : ((nonempty_compacts.Kuratowski_embedding X).val ≃ᵢ
              (nonempty_compacts.Kuratowski_embedding Y).val)
           = ((range (Kuratowski_embedding X)) ≃ᵢ (range (Kuratowski_embedding Y))),
     by { dunfold nonempty_compacts.Kuratowski_embedding, refl },
-  have e' := cast I e,
   have f := (Kuratowski_embedding.isometry X).isometric_on_range,
   have g := (Kuratowski_embedding.isometry Y).isometric_on_range.symm,
-  have h := (f.trans e').trans g,
-  exact ⟨h⟩
+  exact ⟨(f.trans (cast I e)).trans g⟩
```

Comment:
I think it's usually nice to right associate equivs. It should avoid parenthesis accumulation. But that's really minor.
```suggestion
  exact ⟨f.trans $ (cast I e).trans g⟩
```